examples/terraform/openstack/output.tf

```diff
@@ -67,6 +67,8 @@ output "kubeone_workers" {
           # Otherwise, the rootDisk will be on ephemeral storage and its size will
           # be derived from the flavor
           rootDiskSizeGB = 50
+          # Optional: limit how much volumes can be attached to a node
```

Comment:
I haven't found a comment for this field in machine-controller examples, but I assumed it from the field name. Is this comment correct?

Reply:
Yeah, it's optional: https://github.com/kubermatic/machine-controller/pull/584/files#diff-e07452860a04f7019cbb8dfb86d357c1R710